froide_exam/views.py

```diff
@@ -108,3 +119,43 @@ def state_view(request, state_slug=None):
         'types': types,
         'requested_type': requested_type
     })
+
+
+def private_copy(request):
+    token = request.GET.get('token')
+    if token:
+        try:
+            PrivateCopy.objects.get(token=token)
+
+            return render(request, 'froide_exam/view_private_copy.html')
+        except:
+            messages.add_message(request, messages.ERROR,
+                                 _('Ungültiges Token.'))
+
+    if request.method == 'POST':
+        email = request.POST.get('email')
+        try:
+            validate_email(email)
+
+            copy = PrivateCopy.objects.create()
+            url = 'https://fragdenstaat.de/kampagnen/verschlusssache-pruefung/app/privatkopie?token={}'.format(
+                copy.token)
+
+            message = 'Hallo!\r\n\r\nHier kannst du die Prüfungsaufgaben einsehen: {}\r\n\rnBeste Grüße\r\nDas Team von FragDenStaat'.format(
+                url)
+
+            send_mail(
```

Comment:
Das funktioniert. `Froide` hat auf `send_mail` ein [paar eigene APIs](https://github.com/okfde/froide/blob/main/froide/helper/email_sending.py) ([Beispiel-Nutzung](https://github.com/okfde/froide/blob/789ec1893eb345d98ee0c54b43cf68ae0e99a1ae/froide/foirequestfollower/models.py). Damit lassen sich dann in FragDenStaat HTML-E-Mail-Templates für mail intents anlegen. Ist hier aber vielleicht overkill. Einziges gutes Argument: lange Links in Text-E-Mails brechen in manchen Clients um und gehen kaputt. Dafür sind dann HTML-Emails dann ganz gut.

Reply:
habe den link mal auf eine eigene Zeile gepackt und einen kürzeren URL mit Umleitung verwendet, damit das hoffentlich etwas eingedämmt ist